nix/docker/runner

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+filer
+
+export OMNIA_CONFIG=$HOME/omnia.conf
+confer OMNIA > $OMNIA_CONFIG
+
+mkdir -p $HOME/.ssb
+confer SSB > $HOME/.ssb/config
+
+mkdir -p $HOME/logs
+
+ssb-server start 2>$HOME/logs/ssb-server.err >$HOME/logs/ssb-server.out &
```

Comment:
If we're going to write logs to files we probably should think about rotating them as I can potentially see them filling the disk fast.

Reply:
Should probably remove logging to files and just output everything to STDOUT/STDERR. Just wanted to separate `ssb` and `omnia` output. Maybe prefix each output stream with e.g. `[SSB STDERR]`? What do you think, is it overkill?

Reply:
That sounds good to me

Reply:
```
2020-11-03T09:22:49 [OMNIA ERR] [11/03/20 09:22:42] Publishing new price message...
2020-11-03T09:22:49 [OMNIA ERR] [11/03/20 09:22:43] Querying ZRXUSD prices...
2020-11-03T09:22:49 [OMNIA ERR] [11/03/20 09:22:45] [V] bitfinex => 0.3250100000
2020-11-03T09:22:49 [OMNIA ERR] [11/03/20 09:22:45] [V] coinbase => 0.3253000000
2020-11-03T09:22:49 [OMNIA ERR] [11/03/20 09:22:45] [V] binance => 0.3253788120
```

Seeing output being marked as ERR when it isn't (I don't think). This could be an issue with Omnia and beyond this PR though. I wouldn't stress too much about separating ERR and OUT if you ask me. 

This might have something to do with it though:

```
2020-11-03T09:30:54 [OMNIA ERR] jays: error in -e: no such key: tick
2020-11-03T09:30:54 [OMNIA ERR] jays: error in -e: no such key: last
2020-11-03T09:30:54 [OMNIA ERR] [11/03/20 09:30:49] [E] Error - [huobi] Invalid price data = 0.0000000000
2020-11-03T09:30:54 [OMNIA ERR] [11/03/20 09:30:49] [V] uniswap => 0.2893721658
2020-11-03T09:30:54 [OMNIA ERR] [11/03/20 09:30:49] [V] binance => 0.6104736000
2020-11-03T09:30:54 [OMNIA ERR] [11/03/20 09:30:49] [V] kyber => 0.2884828488
2020-11-03T09:30:54 [OMNIA ERR] [11/03/20 09:30:49] [E] Error - [okex] Invalid price data = 0.0000000000
```

Reply:
It's just tagging which lines that are written to `STDERR` and `STDOUT`. It is correct, but maybe just `ERR` is misleading, can change to `STDERR`.